folium/features.py

```diff
@@ -7,10 +7,9 @@
 """
 from jinja2 import Template
 import json
-import base64
 
 from .utilities import color_brewer, _parse_size, legend_scaler, _locations_mirror, _locations_tolist, write_png,\
-    mercator_transform
```

Comment:
We needed this re-factoring!  Thanks!!

@andrewgiessel this is your baby.  Can take a quick look at it before we merge this?